src/components/iconsBackground.jsx

```diff
@@ -0,0 +1,175 @@
+import React from "react";
+import "../styles/Background.scss";
+
+const IconsBackground = () => {
+  return (
+    <React.Fragment>
```

Comment:
Ya quedo viejo React.Fragment ahora se usan empty tags <></>
Para mas info: https://medium.com/fasal-engineering/what-are-react-fragments-or-the-react-empty-tags-190253582905